src/middleware/header.js

```diff
@@ -240,15 +238,34 @@ module.exports = function (middleware) {
 			function (data, next) {
 				async.parallel({
 					scripts: async.apply(plugins.fireHook, 'filter:scripts.get', []),
+					timeagoLocale: (next) => {
+						languages.list(function (err, languages) {
+							if (err) {
+								return next(err);
+							}
+
+							const valid = languages.map(obj => obj.code);
+							if (!valid.includes(res.locals.config.userLang)) {
+								return next();
+							}
+
+							const pathToLocaleFile = '/vendor/jquery/timeago/locales/jquery.timeago.' + utils.userLangToTimeagoCode(res.locals.config.userLang) + '.js';
+							fs.access(path.join(__dirname, '../../public', pathToLocaleFile), function (err) {
```

Comment:
`file.exists`

Reply:
As I understand it, this is here because the `ms` language code exists in NodeBB but not in timeago, so the file doesn't exist. Perhaps we could just special case that instead.